valohai_yaml/objs/pipelines/parameter.py

```diff
@@ -0,0 +1,68 @@
+from typing import TYPE_CHECKING, List, Optional
+
+from valohai_yaml.lint import LintResult
+from valohai_yaml.objs.base import Item
+from valohai_yaml.objs.utils import check_type_and_listify
+from valohai_yaml.types import LintContext, SerializedDict
+
+if TYPE_CHECKING:
+    from valohai_yaml.objs import Config, Pipeline
+
+
+def get_pipeline_parameter_target_node(target: str) -> str:
+    return target.split(".", 1)[0]
+
+
+def get_pipeline_parameter_target_parameter_name(target: str) -> str:
+    return target.split(".", 1)[1]
```

Comment:
Hmm... I can't recall having a spec session about this feature (maybe I was on 🏖️ at that point, who knows), but maybe we should support the full edge target spec for pipeline parameters too? I could easily foresee someone wanting to use a pipeline parameter to replace a node input.

Reply:
I have been thinking about it, how to make sure the targets can target any properties in nodes and your suggestion resolves it elegantly. I will make it a full edge target.